updatehub/tests/successful_integration_test.rs

```diff
@@ -302,6 +302,14 @@ fn correct_config_update_no_polling_probe_api() {
     <timestamp> DEBG staying on Park state.
     "###);
 
+    insta::assert_snapshot!(output_server_info_2.trim(), @r###"
+    <timestamp> INFO installing update: 14aa3cb34ccd7047b81316ba42e5766b8eaeb2ddb32d537856e2819aacdf8a06
+    <timestamp> INFO using installation set as target 1
+    <timestamp> INFO swapping active installation set
+    <timestamp> INFO update installed successfully
+    <timestamp> INFO triggering reboot
+    "###);
```

Comment:
This seems to be changing the code place. Why?

Reply:
I changed this intentionally on next commit, because all the other tests follow the same order: info log then probe log.

Reply:
Good, but then move this to a specific commit as it allows you to add a proper commit log explaining the reason for the move.